README.md

```diff
@@ -4,39 +4,51 @@ This plugin aims to integrate Hibernate Search features to Grails in very few st
 
 - [Grails Hibernate Search Plugin](#grails-hibernate-search-plugin)
   * [Getting started](#getting-started)
-    + [Configuration](#configuration)
-    + [Indexing](#indexing)
-    + [Search](#search)
-    + [Analysis](#analysis)
-    + [Filters](#filters)
-    + [Options](#options)
+  * [Configuration](#configuration)
+  * [Indexing](#indexing)
+    + [Indexing Domain Class](#mark-your-domain-classes-as-indexable)
+    + [Indexing the Data](#indexing-the-data)
+  * [Search](#search)
+    + [Retrieving search results](#retrieving-the-results)
+    + [Mixing with criteria query](#mixing-with-criteria-query)
+    + [Performing SimpleQueryString Searches](#performing-simplequerystring-searches)
+    + [Sorting results](#sorting-the-results)
+    + [Counting results](#counting-the-results)
+    + [Additional features](#additional-features)
+  * [Analysis](#analysis)
+    + [Defining Analyzers](#define-named-analyzers)
+    + [Using Defined Analyzers](#use-named-analyzers)
+  * [Normalizer](#normalizer)
+    + [Defining Normalizers](#define-named-normalizers)
+    + [Using Defined Normalizers](#use-named-normalizer)
+  * [Filters](#filters)
+    + [Defining Filters](#define-named-filters)
+    + [Using Defined Filters](#filter-query-results)
+  * [Options](#options)
+  * [Notes](#notes)
+    + [Updating from 2.2 to 2.3](#updating-from-2.2-to-2.3)
+    + [runtime.groovy vs application.groovy](#runtime.groovy-vs-application.groovy)
+    + [IDE Integration](#ide-integration)
   * [Examples](#examples)
   * [Change log](#change-log)
   * [Authors](#authors)
   * [Development / Contribution](#development-contribution)
   * [License](#license)
 
-
-
 ## Getting started
 
 If you don't want to start from the [template project](#examples), you could start a fresh project:
 
-Add the following repository
-```
-  repositories { maven { url "http://dl.bintray.com/lgrignon/plugins" } }
```

Comment:
Is this not required anymore? I published the plugin on the official repository so I guess this is ok

Reply:
yup, the plugin is available through the standard grails repo